instrumentation/opentelemetry-api/opentelemetry-api-1.0/testing/src/main/java/AgentSpanTestingInstrumenter.java

```diff
@@ -60,16 +76,44 @@ public Request(String name, SpanKind kind) {
 
   private static SpanKey[] getSpanKeys() {
     return new SpanKey[] {
-      SpanKey.SERVER,
+      // span kind keys
+      SpanKey.KIND_SERVER,
+      SpanKey.KIND_CLIENT,
+      SpanKey.KIND_CONSUMER,
+      SpanKey.KIND_PRODUCER,
+      // semantic convention keys
+      SpanKey.HTTP_SERVER,
+      SpanKey.RPC_SERVER,
       SpanKey.HTTP_CLIENT,
       SpanKey.RPC_CLIENT,
       SpanKey.DB_CLIENT,
-      SpanKey.ALL_CLIENTS,
       SpanKey.PRODUCER,
       SpanKey.CONSUMER_RECEIVE,
-      SpanKey.CONSUMER_PROCESS
+      SpanKey.CONSUMER_PROCESS,
     };
   }
 
+  // simulate a real HTTP server implementation
+  private enum HttpServerAttributesExtractor
```

Comment:
maybe `Noop`, `Simple`, `Simple`, etc here too? (avoiding `HttpServerAttributesExtractor` since that is already something)

Reply:
Renamed to `HttpServerSpanKeyAttributesExtractor`